src/templates/example.html

```diff
@@ -34,6 +34,13 @@
         </div>
       </div>
       {{/sections}}
+      <div style="text-align: center">
+        {{#next}}
```

Comment:
Can you put the {{next}} around the div as well? I doesn't make sense to have an empty diff there.